public/docs/_examples/style-guide/ts/01-01/app/kitchen-sink.ts

```diff
@@ -0,0 +1,42 @@
+// #docregion
+/**
+ * Everything in one file ... it's all here!
+ * AVOID THIS PATTERN
+ */
+import { bootstrap } from 'angular2/platform/browser';
+import { Component, OnInit } from 'angular2/core';
+
+class Hero {
+  id: number;
+  name: string;
+}
+
+@Component({
+  selector: 'my-app',
+  template: `
+      <h1>{{title}}</h1>
+      <pre>{{heroes | json}}</pre>
+    `,
+  styleUrls: ['app/app.component.css']
+})
+class AppComponent implements OnInit{
+  title = 'Tour of Heroes';
+
+  heroes: Hero[] = [];
+
+  ngOnInit() {
+    getHeroes().then(heroes => this.heroes = heroes);
+  }
+}
+
+bootstrap(AppComponent, []);
+
+const HEROES: Hero[] = [
+   {id: 1, name: 'Bombasto'},
```

Comment:
Nice catch. There will be a linter on these that @mgechev is adding. It will catch stuff like this.